SparkPost/2web2ui
# SA-1398 - Add report picker and preset reports

### What Changed
 - Added new metrics that the reports need.
 - Added constant file for the new preset reports
 - Added a new component to select saved report.

### How To Test
 - Go to `reports/summary` and select each preset report and check that it shows the correct metrics.
